composer.json

```diff
@@ -17,10 +17,10 @@
   ],
   "require": {
     "php": "^5.6 || ^7.0",
-    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.*"
+    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || 6.0.*"
```

Comment:
**6.0.*** is the equivalent of **>=6.0 <6.1** !

See: https://getcomposer.org/doc/articles/versions.md#wildcard-version-range-

Reply:
Hey @renatomarinho, I suggest replace **6.0.*** for **ˆ6.0**